### ISSUE TITLE
update to minecraft version 1.12 or 1.14 or 1.15

### ISSUE THREAD

**Issue by bluejay94** [microsoft/malmo#860]
current project malmo only supports old version 1.11.2

is it possible to change it up to 1.12 or 1.14 or 1.15?

**Comment by mbrucher** [comment]
That would be indeed great. All the current mods I've seen are on 1.12, 1.14 or 1.15, and with 1.16 arriving, it's too bad malmo is stuck in the past.

**Comment by LinxiFan** [comment]
I agree it's a shame that Malmo doesn't seem to be supported anymore. However, I'm very curious how the authors build the full-featured client for v1.11 ? It looks very similar to the official paid version, unlike the other open-source clients  (https://wiki.vg/Client_List). I wonder how much effort it takes to upgrade the client and associated API to newer versions? I'd really appreciate it if the developers could chime in, thanks! @DaveyBiggers @AndKram
